kbkdf/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "kbkdf"
+version = "0.1.0"
+edition = "2018"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/RustCrypto/KDFs/"
+repository = "https://github.com/RustCrypto/KDFs/"
+description = "Key Derivation Using Pseudorandom Function (KBKDF)"
+keywords = ["crypto", "KBKDF", "KDF"]
+categories = ["cryptography", "no-std"]
+readme = "README.md"
+rust-version = "1.81"
+exclude = ["/tests/data/*"]
```

Comment:
Excluding the test data would break all integration tests, no? If so, we should just exclude `tests/` completely.

Reply:
https://github.com/TheBestTvarynka/KDFs/pull/7

Reply:
done